src/Tile.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react'
+import { Box, Span, theme } from '@truework/ui'
+import styled from 'styled-components'
+import { Field, FieldConfig, FieldProps } from 'formik'
+
+export type TileProps = {
+  id: string
+  icon?: React.ReactElement
+  label: string
+} & React.InputHTMLAttributes<HTMLInputElement>
+
+export type TileFieldProps = { name: string } & TileProps &
+  Pick<FieldConfig, 'validate'>
+
+const TileButton = styled.label(
+  ({ theme }) => `
+    width: 100%;
+    height: 100%;
+    position: relative;
+    display: flex;
+    justify-content: center;
```

Comment:
I don't think you need flex here since the `TileContent` is absolute and sets its own flex props.

```suggestion
    display: block;
```